packages/performance/wafer-map/lighthouserc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    ci: {
+        collect: {
+            staticDistDir: '../dist/wafer-map',
+            numberOfRuns: 1,
+            settings: {
+                onlyAudits: ['user-timings']
```

Comment:
There are a whole bunch of performance metrics that are unnecessary and clutter the output and slow testing time. I referenced the lighthouse docs to show only the user-timings we are capturing